template/functions.php

```diff
@@ -3,4 +3,7 @@ function enqueue_scripts(){
 // css
 wp_enqueue_style('main-css', get_stylesheet_uri());
 }
-add_action('wp_enqueue_scripts', 'enqueue_scripts');
\ No newline at end of file
+add_action('wp_enqueue_scripts', 'enqueue_scripts');
+
+//管理バーの削除
+add_filter( 'show_admin_bar', '__return_false' );
```

Comment:
```suggestion
add_filter('show_admin_bar', '__return_false');
```

[must]
add_actionメソッドと同様に空白はなくしましょう